python/tank/platform/software_launcher.py

```diff
@@ -387,6 +462,16 @@ def version(self):
         return self._version
 
     @property
+    def product(self):
+        """
+        An explicit product name of the DCC represented by this Software Version.
```

Comment:
I fell "An explicit product name FOR the DCC represented" reads better. What say you?